README.md

```diff
@@ -57,7 +57,7 @@ It has the following features, making it useful for developing a trading bot.
 | Binance | ✅ | ✅ | [Link](https://developers.binance.com/docs/binance-spot-api-docs/CHANGELOG) |
 | OKX | ✅ | ✅ | [Link](https://www.okx.com/docs-v5/en/) |
 | Phemex | ✅ | ✅ | [Link](https://phemex-docs.github.io/) |
-| Bitget | ✅ | ✅ | [Link](https://www.bitget.com/api-doc/common/intro) |
+| Bitget | ✅ | ✅ | [Link](https://bitgetlimited.github.io/apidoc/en/mix/) |
```

Comment:
Bitget includes V2 updates as well as website updates.

https://www.bitget.com/support/articles/12560603798900

This PR will not update this and will be issued separately.